packages/edit-navigation/src/components/layout/index.js

```diff
@@ -105,18 +114,26 @@ export default function Layout( { blockEditorSettings } ) {
 								<NavigationEditorShortcuts
 									saveBlocks={ savePost }
 								/>
-								<div
-									className="edit-navigation-layout__canvas"
-									ref={ canvasRef }
-								>
-									<Editor
-										isPending={ ! hasLoadedMenus }
-										blocks={ blocks }
+								{ isMenuDeleted ? (
+									<MenuSelector
+										onSelectMenu={ selectMenu }
+										menus={ menus }
```

Comment:
I think it'd be better to render this in the `EmptyState` component. Maybe we can rename `EmptyState` to `UnselectedMenuState`.

That should already have a lot of the necessary styles (though it will probably need a modifier for the narrower width) and it ensures the options that aren't needed (Select Menu, Add New, Manage Locations, Save) won't be visible:
<img width="670" alt="Screenshot 2021-02-23 at 1 16 33 pm" src="https://user-images.githubusercontent.com/677833/108804878-7a96fa80-75d9-11eb-81f4-3427c80b4594.png">

Reply:
Using EmptyState for those two could be a little bit of overkill. Instead, I have updated MenuSelector to use the `<Card>` component so it shares the common styling. It is a bit different than in a mockup but is consistent with `<EmptyState>`

Reply:
I've added some more detail on why I think using `EmptyState` is the right solution in a comment below:
https://github.com/WordPress/gutenberg/pull/29201#discussion_r586209001